flake-modules/dev/devshell.nix

```diff
@@ -16,7 +16,7 @@
         commands =
           let
             # Thanks to this, the user can choose to use `nix-output-monitor` (`nom`) instead of plain `nix`
-            nix = "$([ '$\{NIXVIM_NOM:-0}' = '1' ] && echo ${pkgs.lib.getExe pkgs.nom} || echo nix)";
+            nix = "$([ $\{NIXVIM_NOM:-0} = '1' ] && echo ${pkgs.lib.getExe pkgs.nix-output-monitor} || echo nix)";
```

Comment:
> You're right `'${VAR:-0}'` is treated as a literal string, however shouldn't we still use `"` quotes to avoid issues if the variable contains whitespace?

Good catch! Fixed

> Also, looking at the `[ ]` test, is `=` correct or should it be `-eq` or `==`?

Looks like there is no difference between them in this case: https://stackoverflow.com/a/20449556/22235705